content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Use Our API to Create an Asset"
+linkTitle: "Create an Asset"
+weight: 110
+description: >
+  Use this sequence of REST calls to create an asset.
+---
+
+{{% pageinfo %}}
+Use this document with our [Cobalt API documentation](https://docs.cobalt.io) to
+manage assets and pentests.
+{{% /pageinfo %}}
+
+<!-- Future task: set up variables for `YOUR-PERSONAL-API-TOKEN` and
+`YOUR-V2-ORGANIZATION-TOKEN`. May support automated populating of REST calls. -->
+
+This page describes how you can use our API To create an [asset](../../getting-started/glossary/#asset). 
+
+## Create an API Token in the Cobalt UI
+
+1. Navigate to https://app.cobalt.io/settings/api-tokens.
+1. If needed, sign in to the app.
+1. Enter an API Token Name.
+1. Select Generate New Token.
+1. In the modal that appears, you should see your API Token, in the **Secret
+   Token** text box.
```

Comment:
1. In the Cobalt app, navigate to **Settings** > [API Tokens](https://app.cobalt.io/settings/api-tokens).
1. Enter an **API Token Name**.
1. Select **Generate New Token**.
1. In the overlay that appears, you should see your API token, in the **Secret
   Token** field.

Reply:
I'd be good with your suggestion in step 1, if we actually had a button with the "Settings" label.

Reply:
IMO, the long link starting with https doesn't look good.
Alternatively, we can describe how to navigate to the page from the user profile.

Reply:
With the alternate solution, I'd want to import the icon associated with settings. (Maybe that'd be better, but that would take some time to research. One of Hugo's weaknesses is that it encourages the use of Bootstrap icons.)

As for the "look" of https://something, we already have many URLs on this page (yes, in code blocks, but still).

Reply:
I'm good with that .